src/main/java/com/abcdedu_backend/post/dto/request/PostCreateRequest.java

```diff
@@ -0,0 +1,31 @@
+package com.abcdedu_backend.post.dto.request;
+
+import com.abcdedu_backend.interceptor.Member;
+import com.abcdedu_backend.board.Board;
+import com.abcdedu_backend.post.Post;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+public class PostCreateRequest {
+    private String title;
+    private String content;
+    private Long viewCount;
+    private Boolean secret;
+    private Boolean commentAllow;
+
+    private String boardName;
+
+    public Post toEntity(Member member, Board board) {
+        return Post.builder()
+                .board(board)
+                .member(member)
+                .title(this.title)
+                .content(this.content)
+                .viewCount(this.viewCount)
+                .secret(this.secret)
+                .commentAllow(this.commentAllow)
+                .build();
+    }
```

Comment:
이부분은 사소한 부분인데요, 고민하다가 저는 이런 Dto toEntity함수를 request에서 entity를 알지 않아도 된다고 생각해서 service에서 작성했었습니다. 단점으로는 service에서 너무 로직이 많아질 수 있을 것 같아요. Mapper도 사용한다고 하는데, 굳이라는 생각도 드는것 같습니다. request에서 하게 되면 toEntity()로 바로 편리하게 사용할 수 있을 것 같아요.하나로 통일하면 좋을 것같은데 어떤 방식으로 진행하면 좋을까요?
```suggestion
    private Member createMember(SignUpRequest request) {
        Member signUpMember = Member.builder()
                .name(request.name())
                .email(request.email())
                .encodedPassword(passwordEncoder.encode(request.password()))
                .build();
        return signUpMember;
    }
```

Reply:
말씀하신대로.. 책임분리가 안된다는 단점이 있긴 합니다.  설계 생각하면 서비스 계층에서 변환하는게 맞아요,,! 제가 수정해서 올리겠습니다.